### PR TITLE
Update pe.m3u

### PR DESCRIPTION
**Pull request by panctvtucuman**
Update pe.m3u #16386

### CODE REVIEW

**Comment by panctvtucuman** [comment]
agregar #16386

File: streams/pe.m3u [hunk 1]
```diff
@@ -157,6 +157,8 @@ http://cdn2.ujjina.com:1935/iptvovacion1/liveovacion1tv/playlist.m3u8
 https://5c3fb01839654.streamlock.net:1963/iptvovacion1/liveovacion1tv/playlist.m3u8
 #EXTINF:-1 tvg-id="PanamericanaTV.pe",Panamericana TV (1080p) [Not 24/7]
 https://cdnhd.iblups.com/hls/ptv5.m3u8
+#EXTINF:-1 tvg-id="PeruPanc.Pe",Peru Panc TV (1080p)
```
Comment (BellezaEmporium) [inline streams/pe.m3u:160]:
```suggestion
#EXTINF:-1 tvg-id="PeruPancTV.pe",Peru Panc TV (1080p)
```

**Review by BellezaEmporium — APPROVED** [review]

**Comment by BellezaEmporium** [comment]
Database will need to be updated.

We have a "PancTV.ar" that exists, in San Miguel de Tucuman, though this variant doesn't exist in our database.

**Review by Dum4G — APPROVED** [review]

**Review by BellezaEmporium — APPROVED** [review]

**Comment by BellezaEmporium** [comment]
A suggestion by @ADPNetworkPlus prevents me from going further.

**Review by Sphinxroot — APPROVED** [review]

**Pull request closed without merging**